### DISCUSSION TITLE
Broken Images

### DISCUSSION THREAD

**Discussion by moonrivervoyager** [community/community#32341] [Other Feature Feedback, Questions, & Ideas]
Hi everyone! I'm pretty new to both coding and GIthub. Could someone help me figure out why my images aren't displaying? I checked capitalization and the folders and the other typical things that can cause broken images. Have I pushed files to the wrong place? 
Thanks so much!
https://github.com/moonrivervoyager/teacozy

**Comment by airtower-luna** [comment]
I'm afraid you didn't check folders closely enough, [for example here](https://github.com/moonrivervoyager/teacozy/blob/1b52ac70f07fec2537d5f86e8de1f6366c83644d/index.html#L11):

```html
            <img src="../teacozyproject/img-tea-cozy-logo.png" alt="Tea Cozy Logo" />
```

This looks for the picture one directory up and then in a directory called `teacozyproject`, when your pictures are actually in the same directory. This would work (but be bad practice) if the repository name (and thus the path of the deployed site) was exactly that, but it isn't.

What you should do is simply refer to images in the same directory by name, like so:

```html
            <img src="img-tea-cozy-logo.png" alt="Tea Cozy Logo" />
```

**Reply by moonrivervoyager** [reply]
Thank you so much! That really helps!

**Reply by airtower-luna** [reply]
Glad to hear! Marking the answer would be appreciated. :smiley_cat:
